src/pages/contact-us.js

```diff
@@ -0,0 +1,236 @@
+import React from 'react'
+
+import Layout from '../components/Layout'
+import ConstrainedWidth from '../components/Layout/ConstrainedWidth'
+
+import Text from '../components/Text'
+
+import VerticalSpacing from '../components/VerticalSpacing'
+import PageMeta from '../components/PageMeta'
+
+import Container from '../components/Container'
+import Twitter from '../components/twitter.svg'
+import PageTop from '../components/PageTop'
+import contactArcs from '../components/contactArcs.svg'
+import EmailLink from '../components/EmailLink'
+const ContactUs = () => (
+  <Layout>
+    <PageMeta
+      title="Contact us"
+      description="To get in touch, you can contact us via email or twitter."
+    />
+    <PageTop>
+      <ConstrainedWidth>
+        <Text size="xlarge" color="#48e9ce">
+          <h1>Get in touch</h1>
+        </Text>
+
+        <div className="contact-header">
+          <Text size="normal">
+            <div className="subtitle">
+              Drop us an email. We’d love to have a conversation about how we
+              could work with you.
+            </div>
+          </Text>
+          <div className="main-arc-container">
+            <img src={contactArcs} height={136} width={191} />
+          </div>
+        </div>
+
+        <EmailLink
+          link="/"
+          background="#5600ee"
+          border_color="#5600ee"
+          mobileSize="0.9rem"
+        >
+          Email: hello@neontribe.co.uk
+        </EmailLink>
+      </ConstrainedWidth>
+    </PageTop>
+    <VerticalSpacing size={5} />
+
+    <ConstrainedWidth>
+      <Text size="normal" color="black" maxCharacter="49ch">
+        Follow us on Twitter
+      </Text>
+
+      <div className="social-media-container">
+        <div className="social-media-labels">
+          <img src={Twitter} height={42} width={42}></img>
+
+          <Text color="#561dee" size="normal">
+            @neontribe
+          </Text>
+        </div>
```

Comment:
Should there be a link to Twitter here? "Follow us on Twitter" sounds like a call to action, but there isn't anything to interact with. We can also help out screen readers with an `alt` attribute on the image.